docs/journal/01-cold-storage.md

```diff
@@ -1,74 +1,90 @@
 
-### Build a cloud service that receives events posted as HTTP requests
+Notice that in this design, the HTTP service handles both ingestion and processing.
 
-The cloud service could be implemented as a web role that parses and saves the event information to cold storage. The cloud service effectively performs the ingestion and processing steps described in [Introducing the Journey][00-intro]
+![cloud service solution based on web roles](media/01-cold-storage/physical-architecture-cloud-service.png)
 
 **Pros:**
-- This approach is technically straightforward and quick to prototype
+
+- Technically straightforward and quick to prototype.
 
 **Concerns:**
-- **Throughput.** All incoming requests are handled by the HTTP.sys driver (the entry point into the IIS services stack), and are queued for processing by the IIS service. Worker threads running application code pick up these requests and handle them. The thread pool must be of sufficient size to prevent requests from backing up in the IIS queue (and possibly timing out, causing the request to be rejected), and the processing performed must run quickly enough to satisfy the time requirements for recording event information. To reduce these concerns, it is possible to scale the system by adding more instances. The Azure load-balancer can transparently direct requests to ensure an even distribution. However, there is a limit of 25 role instances per deployment. For a system of the size anticipated by the customer this should be fine, but other customers may have requirements that require many more instances; Fabrikam don't want to have to radically redesign the system in the future.
 
-> ![Poe](media/PersonaPoe.png) 
-"We briefly considered building a set of custom virtual machines running an open source web server in the cloud to see whether we could optimize it specifically for this task. We know that you can distribute the load across VMs by configuring load balancing with Azure. However, I asked the developers whether they really wanted Fabrikam to have to take responsibility for maintaining and patching these VMs and the web server software. Additionally, Gary was concerned about the security aspects of this approach - hardening a custom web server is a complex task. We decided against this approach pretty quickly."
+- **Throughput.** The system must avoid request timeouts. IIS puts requests in a queue until a worker thread running application code picks them up. The thread pool must be large enough so that requests don't back up in the queue, possibly timing out. You can scale by adding more instances, and the Azure load-balancer will transparently direct requests to ensure an even distribution. However, there is a limit of 25 role instances per deployment. That should be enough for the Fabrikam scenario, but may not be enough for other scenarios.
```

Comment:
I think that **Throughput** and **Scalabity** are very closely related. Perhaps we should collapse the two points?
Often we'll use _Throughput_ as a measure of the density on single node and _scalabity_ for about adding new nodes. Still, overall system _Throughput_ is both combined.